rpc/internal/perf_service_test.go

```diff
@@ -345,8 +384,8 @@ func TestAttachResultData(t *testing.T) {
 			defer func() {
 				require.NoError(t, tearDownEnv(env, false))
 			}()
-			port := getPort()
 
+			port := getPort()
```

Comment:
?

Reply:
Just moved it down a line lol. It felt like it belonged with this code block.

Reply:
Ok so it was intentional, good.